docs/testkube-pro/articles/environment-management.md

```diff
@@ -1,19 +1,21 @@
 # Environment Management
 
-Environment in Testkube is related to the Testkube agent, which is responsible for sending
-test insights to Testkube Pro and for managing your Kubernetes related cluster resources.
+:::note
+An environment is a grouping of users, workflows, results, triggers, and webhooks. A Testkube environment is managed by a Testkube Agent running in your Kubernetes Cluster - read more [here](../articles/architecture.md).
```

Comment:
almost :D - the "An environment is a grouping of users, workflows, results, triggers, and webhooks." statement should be before/outside the note

Reply:
Sorry - done.